product_docs/docs/eprs/7/07_common_operations/09_offline_snapshot.mdx

```diff
@@ -87,26 +88,33 @@ The default value is `true`.
     -   Set the `batchInitialSync` option to the appropriate setting for your situation as discussed at the end of [Non-batch mode synchronization](#non_batch_mode_sync).
 
 1.  If you modified the publication server configuration file in Step 3, reload configuration of the publication server. See ["Reloading the Publication or Subscription Server Configuration File (reloadconf)"](../05_smr_operation/02_creating_publication/01_registering_publication_server/#registering_publication_server) for directions on reloading the publication server's configuration.
 
-1.  In the subscription database, create the schema and the subscription table definitions, and load the subscription tables from your offline data source. The subscription database user name used in [Adding a subscription database](../05_smr_operation/03_creating_subscription/02_adding_subscription_database/#adding_subscription_database) must have full privileges over the database objects created in this step. Also review the beginning of [Adding a subscription database](../05_smr_operation/03_creating_subscription/02_adding_subscription_database/#adding_subscription_database) regarding the rules as to how Replication Server creates the subscription definitions from the publication for each database type. You must follow these same conventions when you create the target definitions manually.
+### Creating subscription servers 
+
+Execute these steps to create a subscription from an offline snapshot. Repeat them for each additional subscription. 
 
 1.  Add the subscription as described in [Adding a subscription](../05_smr_operation/03_creating_subscription/03_adding_subscription/#adding_subscription).
 
+1.  In the subscription database, create the schema and the subscription table definitions, and load the subscription tables from your offline data source. The subscription database user name used in [Adding a subscription database](../05_smr_operation/03_creating_subscription/02_adding_subscription_database/#adding_subscription_database) must have full privileges over the database objects created in this step. Also review the beginning of [Adding a subscription database](../05_smr_operation/03_creating_subscription/02_adding_subscription_database/#adding_subscription_database) regarding the rules as to how Replication Server creates the subscription definitions from the publication for each database type. You must follow these same conventions when you create the target definitions manually.
+
+    !!!note 
+    Ensure you synchronize the offline data source from the source Publication database after you have completed the creation of a subscription. Otherwise, certain changes from the source database won't be replicated.
```

Comment:
Since the term "synchronize" is specifically used for replication of incremental (CDC) changes in EPRS cluster, so I would suggest to use "populate" (or an equivalent) i.e. "Ensure you populate the offline data source from the source...", so that end user is not confused.